tests/mockdata/adq.py

```diff
@@ -0,0 +1,75 @@
+
+import numpy as np
+
+from extra_data.tests.mockdata.base import DeviceBase
+
+from extra.utils import gaussian
+
+
+class AdqDigitizer(DeviceBase):
+    # still needed?
+    control_keys = [
+        ('fakeScalar', 'u4', ()),
+    ]
```

Comment:
Is there nothing really in the CONTROL section?

Reply:
Currently nothing of interest or that is used. The interesting values are only in the `RUN` section, in particular after we recently started using property filtering to work around serious aggregator issues when saving all `CONTROL` values.

Reply:
I assume you've put `fakeScalar` here because something goes wrong if it's an empty list? Is that something we should fix, or not worth the effort?

Reply:
I documented here a while ago, but did not get back to it yet: https://github.com/European-XFEL/EXtra-data/issues/429

Reply:
Aha, thanks, I missed that.